nuxeo-common/src/main/java/org/nuxeo/common/utils/TextTemplate.java

```diff
@@ -261,7 +261,7 @@ public Properties preprocessVars(Properties unprocessedVars) {
             }
             recursionLevel++;
             // Avoid infinite replacement loops
-            if ((!doneProcessing) && (recursionLevel > MAX_RECURSION_LEVEL)) {
+            if (!doneProcessing && recursionLevel > MAX_RECURSION_LEVEL) {
```

Comment:
Please keep cleanups in separate commits (i.e., don't have your IDE automatically do cleanups).